vent/service/consumer.go

```diff
@@ -67,7 +67,8 @@ func (c *Consumer) Run() error {
 
 	c.Log.Info("msg", "Connecting to SQL database")
 
-	db, err := sqldb.NewSQLDB(c.Config.DBURL, c.Config.DBSchema, c.Log)
+	//TODO: implement also for SQLITE
+	db, err := sqldb.NewSQLDB("postgres", c.Config.DBURL, c.Config.DBSchema, c.Log)
 	if err != nil {
```

Comment:
"postgres" should be a parameter (named "db-adapter" for instance) with a list of actual supported dbms, in this case only postgres
in `ventCmd.Flags().StringVar(&cfg.DBAdapter, "db-adapter", cfg.DBAdapter, "Database adapter (supported dbms=postgres)")`

then 
`db, err := sqldb.NewSQLDB(c.Config.DBAdapter, c.Config.DBURL, c.Config.DBSchema, c.Log)`